<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherHomepage</class>
 <widget class="QWidget" name="LauncherHomepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 247, 247);</string>
  </property>
  <widget class="QLabel" name="label_recentPJ">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>85</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>最近使用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_grid">
   <property name="geometry">
    <rect>
     <x>834</x>
     <y>78</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Images/MainWindowGrid2_Bright.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_list">
   <property name="geometry">
    <rect>
     <x>904</x>
     <y>78</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Images/MainWindowList1_Bright.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_sort">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>128</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>133</y>
     <width>1</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(218, 218, 218);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_sort">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>132</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>   名称</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   时间</string>
    </property>
   </item>
  </widget>
  <widget class="QScrollArea" name="main_scrollArea">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>175</y>
     <width>951</width>
     <height>495</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>951</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>881</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(218, 218, 218);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>3</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎使用 Cell WorkStation。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
